en-US/main.lang

```diff
@@ -817,3 +817,39 @@ Create account
 # Used in a Firefox Account widget, button label to download a mobile app
 ;Get the app
 Get the app
+
+
+;Follow @%s
+Follow @%s
+
+
+;Follow
+Follow
+
+
+;Follow %s on Twitter
+Follow %s on Twitter
+
+
+;Twitter Timeline of %s
+Twitter Timeline of %s
+
+
+;Reply
+Reply
+
+
+;Retweet
+Retweet
+
+
+;Favorite
+Favorite
+
+
+;<strong>Enter your email</strong> to access Firefox Accounts.
```

Comment:
Remove all new strings above this, they're not part of [this range](https://github.com/jpetto/bedrock/blob/4a4da9d750695cc0c592690773d9874aa43438c5/bedrock/base/templates/macros.html#L410-L441).